ug-vmware-users/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at http://git.k8s.io/community/contributors/guide/owners.md
+
+reviewers:
+  - ug-vmware-users-leads
+approvers:
+  - ug-vmware-users-leads
+labels:
+  - ug/vmware
```

Comment:
(non-blocking nit for the charter) I guess the label should be `ug/vmware-users` to stay consistent with the nomenclature?